### PR TITLE
refactor(app/renderer): remove containers

### PR DESCRIPTION
**Pull request by adnasa**
it's time we remove `containers`
all of this is to prepare the usage of the new (now kinda old) context

- ~~blocked by #222 (cherry picked)~~

### CODE REVIEW

File: app/renderer/src/components/app/app.js [hunk 1]
```diff
@@ -24,68 +31,132 @@ const base = style({
   overflow: 'hidden',
 });
 
-export default function App(props) {
-  const outerStyles = props.theme.window
+const getOuterStyles = theme =>
+  theme && theme.window
```
Comment (adnasa) [inline app/renderer/src/components/app/app.js]: urgh, we really ought to remove this.. soon! :puke:

**Comment by adnasa** [comment]
I've refactored all I could manage, as for `result-item.js`, I really don't understand this. Can you take over, @vutran ?

**Review by vutran — APPROVED** [review]
LGTM

**Comment by adnasa** [comment]
I would like to have `result-item.js` migrated before I merge this

**Pull request merged**